src/nvim/ex_getln.c

```diff
@@ -2398,6 +2398,13 @@ static void cmdpreview_show(CommandLineState *s)
     emsg_off--;
     goto end;
   }
+
+  // Command is invalid, return.
+  // This check can be removed if there's ever a preview function added for :filter.
```

Comment:
Just a small advice, I'd try to create a test that includes the repro from https://github.com/neovim/neovim/issues/19057 and validates whatever behavior you decided is the valid outcome, in case someone else (or even future you) misses this comment (which may get outdated) for one reason or another.